packages/blockchain-api/package.json

```diff
@@ -32,12 +32,9 @@
     "@google-cloud/nodejs-repo-tools": "^2.3.3",
     "@types/dotenv": "^4.0.3",
     "@types/express": "^4.16.1",
-    "@types/jest": "^24.0.17",
     "@types/utf8": "^2.1.6",
     "@types/web3": "^1.0.18",
-    "jest": "^24.8.0",
```

Comment:
Did you remove these because they're already in the root package.json? If so, why not do the same for typescript below?